### PR TITLE
updating rails-5 branch to foundation 6.4

### PR DESCRIPTION
**Pull request by sineade**
:elephant:

This is to update foundation version 6.4.1.2

https://amaabca.visualstudio.com/dx_technical_debt/_workitems?id=926&_a=edit

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [x] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [x] I've added new components to the style guide (or have a PR in to add them).
- [x] Any applicable version numbers have been updated.
- [x] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
~~- [ ] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).~~
- [x] I have written a detailed PR message explaining what is being changed and why
- [x] I’ve linked to any relevant VSTS tickets
- [x] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

**Comment by sineade** [comment]
:hankey: not ready for review

**Comment by sineade** [comment]
:wave: ready for review

**Comment by mvandenbeuken** [comment]
📖

File: lib/ama_layout/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = '6.9.0.pre'
+  VERSION = '6.9.1.pre'
```
Comment (mvandenbeuken) [inline lib/ama_layout/version.rb]: would you consider this a bug fix, or a minor bump?

Bug fix would be 6.9.1.pre, minor would be 6.10.0.pre.

Comment (sineade) [inline lib/ama_layout/version.rb]: ah yes, a minor bump.

**Comment by sineade** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
📖

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**